tsconfig.json

```
@@ -21,7 +21,7 @@
 
         // Stricter Typechecking Options
         "noUncheckedIndexedAccess": true,
-        "exactOptionalPropertyTypes": true,
+        // "exactOptionalPropertyTypes": true,
```

Rules which are commented out should show the default value:
```suggestion
        // "exactOptionalPropertyTypes": false,
```